contrib/win32/win32compat/misc.c

```diff
@@ -314,22 +363,35 @@ spawn_child(char* cmd, int in, int out, int err, DWORD flags) {
 	si.dwFlags = STARTF_USESTDHANDLES;
 
 	b = CreateProcessW(NULL, cmd_utf16, NULL, NULL, TRUE, flags, NULL, NULL, &si, &pi);
+	GOTO_CLEANUP_IF(!b, ENOEXEC);
+	CloseHandle(pi.hThread);
```

Comment:
If **Thread Handle** not using after **CreateProcess**, then need close this handle!!!
Function sw_remove_child_at_index close only **Process Handle** (proof: https://github.com/PowerShell/openssh-portable/blob/fb511200930e9f78481154215aab8a7e5e1df5d9/contrib/win32/win32compat/signal_sigchld.c#L77)